agent/README.md

```diff
@@ -501,8 +501,6 @@ node ./main.js --verbose=9 --expose --colors \
 --nodes-count=value.............S-Chain nodes count.
 --time-framing=value............Specifies period(in seconds) for time framing. Zero means disable time framing.
 --time-gap=value................Specifies gap(in seconds) before next time frame.
---ptx...........................Enable pending transaction analysis to avoid transaction conflicts.
```

Comment:
`--ptx` turns on pending transaction analysis subsystem. When each `n-th` IMA Agent queries `(n-1)th, (n-2)th, ...` whether it completed working. Needed to avoid `revert`s on submitting same IMA messages twice.

Reply:
It's not removed. It's renamed to `--pwa` and turned on by default and we never need to turn it off on real nodes.

Reply:
Please add it to the README

Reply:
Fixed